src/views/Login.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="max-w-lg mx-auto p-4 shadow-lg mt-6">
+    <form @submit.prevent="login" class="p-8 border rounded">
+      <div class="flex items-center text-2xl justify-center mb-4">
+        <img
+          class="w-16 h-16 mr-4 rounded-full"
+          src="@/assets/tu20logo.png"
+          alt="TU20 Logo"
+        />
+        TU20 Bot Config
+      </div>
+
+      <div>
+        <small class="font-light text-gray-500 text-xs mb-2"> Username </small>
+        <input
+          type="text"
+          class="border rounded p-2 text-lg w-full bg-gray-100 font-mono"
+          v-model="username"
+          required
+        />
+        <small class="font-light text-gray-500 text-xs mb-2 mt-4">
```

Comment:
Looking at the image, I would increase top margin here.
```suggestion
        <small class="font-light text-gray-500 text-xs mb-2 mt-8">
```